### PR TITLE
Localization Hindi India

### PR DESCRIPTION
**Pull request by SensehacK**
- [x] This change has been discussed with the author.
- [x] It implements / fixes issue #2054 

---

**Description:**

Have added Localization Translation support for Hindi Language - India.

### CODE REVIEW

**Comment by SensehacK** [comment]
Resolved Conflicts with two files :)

**Comment by SensehacK** [comment]
I didn't know what to upload with IINA Translation utility, But I'm attaching this screenshot with the utility.
Please let me know If I'm missing anything. Happy to contribute more :)

screenshot 2019-01-26 at 12 43 59

File: iina/hi.lproj/PrefGeneralViewController.strings [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+"jeK-46-2BG.title" = "इतिहास:";
+
+/* Class = "NSButtonCell"; title = "Play when entered full screen"; ObjectID = "lOw-pu-Llm"; */
+"lOw-pu-Llm.title" = "जब पूर्ण स्क्रीन में प्रवेश किया है तब खेलें";
+
+/* Class = "NSButtonCell"; title = "Include subtitles"; ObjectID = "lvA-Rc-Sh6"; */
+"lvA-Rc-Sh6.title" = "उपशीर्षक शामिल करें";
+
+/* Class = "NSTextFieldCell"; title = "Format:"; ObjectID = "oJY-Iz-3sO"; */
+"oJY-Iz-3sO.title" = "Format:";
```
Comment (alejx) [inline iina/hi.lproj/PrefGeneralViewController.strings]: Is this string missed?

File: iina/hi.lproj/PrefSubViewController.strings [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+"1Tq-5k-o6R.title" = "टॉप";
+
+/* Class = "NSButtonCell"; title = "Display subtitles in letterboxes while in full screen"; ObjectID = "2XY-Un-0Fg"; */
+"2XY-Un-0Fg.title" = "पूर्ण स्क्रीन में रहते हुए लेटरबॉक्स में उपशीर्षक प्रदर्शित करें";
+
+/* Class = "NSMenuItem"; title = "OpenSubtitles.org"; ObjectID = "39k-VH-tOz"; */
+"39k-VH-tOz.title" = "OpenSubtitles.org";
+
+/* Class = "NSTextFieldCell"; title = "strip"; ObjectID = "3H8-Ei-cTm"; */
+"3H8-Ei-cTm.title" = "strip";
```
Comment (alejx) [inline iina/hi.lproj/PrefSubViewController.strings]: Ditto.

File: iina/hi.lproj/PrefNetworkViewController.strings [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+"2qf-cC-Lfi.title" = "कस्टम youtube-dl पथ:";
+
+/* Class = "NSTextFieldCell"; title = "Default: 153600"; ObjectID = "3os-7u-lv8"; */
+"3os-7u-lv8.title" = "डिफ़ॉल्ट: 153600";
+
+/* Class = "NSMenuItem"; title = "TCP"; ObjectID = "6JF-48-aCE"; */
+"6JF-48-aCE.title" = "TCP";
+
+/* Class = "NSTextFieldCell"; title = "HTTP proxy:"; ObjectID = "9ut-ID-ffZ"; */
+"9ut-ID-ffZ.title" = "HTTP proxy:";
```
Comment (alejx) [inline iina/hi.lproj/PrefNetworkViewController.strings]: Ditto

Comment (SensehacK) [reply iina/hi.lproj/PrefNetworkViewController.strings]: @alejx Hi TCP & HTTP would stay in english right as their is no hindi equivalent & would it be considered noun.

File: iina/hi.lproj/PrefGeneralViewController.strings (hunk 1 above)
Comment (SensehacK) [reply iina/hi.lproj/PrefGeneralViewController.strings]: Done 👍

File: iina.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -2037,7 +2037,7 @@
 				sk,
 				sv,
 				ro,
-        hi,
+        		hi,
```
Comment (alejx) [inline iina.xcodeproj/project.pbxproj]: What did you do in this commit? It seems not right viewing on GitHub.

Comment (SensehacK) [reply iina.xcodeproj/project.pbxproj]: When I pressed resolve conflict issues on Github Web, It doesn't indents properly according to Xcode indentation. So I had to take pull on my local machine & add proper indentation in "project.pbxproj" file using Visual Studio Code editor.

Comment (alejx) [reply iina.xcodeproj/project.pbxproj]: But still, it looks not correct now. Can you try to fix that again?

Comment (SensehacK) [reply iina.xcodeproj/project.pbxproj]: So on github Web it shows different indentation, should I just revert my indentation changes ?

Comment (SensehacK) [reply iina.xcodeproj/project.pbxproj]: Attaching a screenshot :) 
On github Desktop & Xcode / VS Code looks fine. 
screenshot 2019-01-31 at 20 45 07

Comment (alejx) [reply iina.xcodeproj/project.pbxproj]: Your previous version used all spaces; current version used mixed spaces and tabs. Only tabs is used in other lines.

Comment (SensehacK) [reply iina.xcodeproj/project.pbxproj]: Please suggest me what should I use to check whether the indentation is correct. 

Now using VS Code for deleting whitespace / Tabs & then pressing enter.

screenshot 2019-01-31 at 21 05 37

**Pull request merged**